airbyte-integrations/connectors/source-google-search-console/source_google_search_console/spec.py

```diff
@@ -0,0 +1,148 @@
+#
```

Comment:
shouldn't we drop spec.json?

Reply:
json was moved to `integration_tests` folder